hw/dv/data/tests/lock_reg_test.hjson

```diff
@@ -0,0 +1,10 @@
+{
+  tests: [
+    {
+      name: "{name}_lock_reg_test"
```

Comment:
do you guys think we should actually call this `enable_reg` test?
i know the lock name has probably been perpetuated by me...but since the RTL actually implements the opposite (enable function), i wonder if we should actually just be consistent.

Reply:
Thanks for the suggestion, I think be consistent with RTL is good. I will update it :)

Reply:
How about `regen_test`? Somehow feel like `enable_reg_test` sounds like this test will only enable a register. Feel free to provide any feedback, as I am really bad at naming...